src/components/Items/ProfileContact.js

```diff
@@ -37,7 +37,7 @@ const ProfileContact = ({
     <Pressable
       onPress={onPress}
       android_ripple={{
-        color: COLORS.lightgrey,
+        color: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color for the ripple effect on Android has been changed from `COLORS.lightgrey` to `COLORS.gray100`. Ensure that this color change is consistent with the overall theme and visibility of the ripple effect.
```diff
-        color: COLORS.lightgrey,
+        color: COLORS.gray100,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->